### PR TITLE
Add more tooling

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
Updated to clean Ubuntu 12.04 too.

**Comment by bastelfreak** [comment]
Should we clean fedora 24 and older as well?

**Comment by ekohl** [comment]
Makes sense.

**Comment by ekohl** [comment]
Updated to prune Fedora and Red Hat too.

File: bin/clean-metadata [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import os
+import json
+
+# Note range is exclusive so the last number is not in the list
+UNSUPPORTED = {
+    'CentOS': [str(i) for i in range(3, 6)],
+    'Debian': [str(i) for i in range(3, 6)],
+    'Fedora': [str(i) for i in range(3, 25)],
```
Comment (bastelfreak) [inline bin/clean-metadata]: Fedora 25 still has support. 24 and older is dead.

Comment (ekohl) [reply bin/clean-metadata]: Note the comment above: range is exclusive so 25 is not included.

Comment (bastelfreak) [reply bin/clean-metadata]: ah ups. but then we should increase debian to 7 (or even 8)?

Comment (ekohl) [reply bin/clean-metadata]: You're right. 8 it is.

**Comment by ekohl** [comment]
Expanded to prune all unsupported Ubuntu releases:
```
Reading modules/puppet-logrotate/metadata.json
Removing CentOS-5
Removing Debian-6
Removing Debian-7
Removing Ubuntu-12.04
Removing Ubuntu-10.04
Writing modules/puppet-logrotate/metadata.json
```

**Comment by bastelfreak** [comment]
Thanks for the work @ekohl!

**Pull request merged**